src/tests/unit/tests/DetailsView/reports/components/assessment-report-step-list.test.tsx

```diff
@@ -23,8 +22,8 @@ describe('AssessmentReportStepListTest', () => {
             steps: AssessmentReportBuilderTestHelper.getRequirementReportModelPass(),
         };
 
-        const shallow = shallowRender(<AssessmentReportStepList {...props} />);
-        expect(shallow).toMatchSnapshot(); //
+        const wrapper = shallow(<AssessmentReportStepList {...props} />);
+        expect(wrapper.getElement()).toMatchSnapshot(); //
```

Comment:
Any reason for this '//' here (and in the file)?

Reply:
:( not really... didn't even see it really.

Reply:
updated